internal/supervisor/import.go

```diff
@@ -138,19 +138,26 @@ func (s *ImportSupervisor) Do() error {
 	if err != nil {
 		s.logger.WithError(err).Errorf("Failed to perform work on Import %s", work.ID)
 		workError := err.Error()
+
 		go func() {
-			for {
+			completeAt := model.GetMillis()
+
+			expBackoff := utils.NewExponentialBackoff(time.Second*5, time.Minute*10, time.Minute*30)
+			err := expBackoff.Retry(func() error {
 				err := s.awatClient.CompleteImport(
 					&awat.ImportCompletedWorkRequest{
 						ID:         work.ID,
-						CompleteAt: model.GetMillis(),
+						CompleteAt: completeAt,
 						Error:      workError,
 					})
-				if err == nil {
-					return
+				if err != nil {
+					s.logger.WithError(err).Errorf("failed to report error to AWAT for Import %s", work.ID)
```

Comment:
Just to know if it failed the first time on under a retry (though we'll see multiple loglines, I think this may be more clear?) 0/5

```suggestion
					s.logger.WithError(err).Errorf("failed retry to report error to AWAT for Import %s", work.ID)
```